client-templates/bitbucket-server/accept.json.sample

```diff
@@ -526,6 +524,17 @@
         "password": "${BITBUCKET_PASSWORD}"
       }
     },
+    {
+      "//": "get list of API capabilities",
+      "method": "GET",
+      "path": "/rest/capabilities",
```

Comment:
Added for
https://github.com/snyk/bitbucket-server-agent/blob/1454badf76fa7f12bb8d80760bd2a396c047606d/lib/bitbucket-server/codeinsights.js#L96